recipes/logstash.rb

```diff
@@ -50,15 +38,15 @@
 unless node.run_state['lumberjack_decoded_certificate'].nil? || node.run_state['lumberjack_decoded_certificate'].nil?\
  || node['elkstack']['config']['agent_protocol'] != 'lumberjack'
   node.default['elkstack']['config']['logstash']['server']['my_templates']['input_lumberjack'] = 'logstash/input_lumberjack.conf.erb'
-  template_variables['input_lumberjack_ssl_certificate'] = node['lumberjack']['ssl_cert_path']
-  template_variables['input_lumberjack_ssl_key'] = node['lumberjack']['ssl_key_path']
+  node.default['elkstack']['config']['logstash']['server']['my_template_variables']['input_lumberjack_ssl_certificate'] = node['lumberjack']['ssl_cert_path']
+  node.default['elkstack']['config']['logstash']['server']['my_template_variables']['input_lumberjack_ssl_key'] = node['lumberjack']['ssl_key_path']
```

Comment:
I thought we had to set a precedence level when defining an attribute in a recipe.